### ISSUE TITLE
Urgent: Remove "Index of /___" Pages

### ISSUE THREAD

**Issue by jeromew21** [pioneers/website#497]
https://pioneers.berkeley.edu/compete/

Yikes.

**Comment by jeromew21** [comment]
This is related to the earlier issue about serving the site as a static site with the ruby dev server.

**Comment by kev-chien** [comment]
So if we change it to production mode this will disappear?

Also, I think we could look into redirecting such URLs, I believe there were some examples in the repo. But it would be something to move away from in the long-term.

**Comment by jeromew21** [comment]
Yeah, switching to a production server would probably fix this, but we should also look into redirects.
